change/@fluentui-react-c61ebf99-c8f7-4f1f-9e55-73309b76987b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This should be minor since it's adding an API
```suggestion
  "type": "minor",
```